rllib/policy/policy.py

```diff
@@ -272,27 +290,29 @@ def export_checkpoint(self, export_dir):
         """
         raise NotImplementedError
 
+    @staticmethod
+    def clip_action(action, space):
```

Comment:
This doesn't need to be a class member

Reply:
Will change back, but the `clip_actions` belongs either into utils. Or: If only Policies ever use it, then yes, it should be a staticmethod.

Reply:
Moved back to being a function.